src/libclipper/src/datatypes.cpp

```diff
@@ -141,6 +143,8 @@ size_t DoubleVector::serialize(uint8_t *buf) const {
 }
 
 size_t DoubleVector::hash() const {
+  // TODO [CLIPPER-63]: Find an alternative to hashing doubles directly,
+  // as this is generally a bad idea due to comparison difficulties
```

Comment:
s/due to comparison difficulties/due to loss of precision from floating point representation